charts/kof-collectors/templates/opentelemetry/collector.yaml

```
@@ -37,18 +37,17 @@ spec:
 
     processors:
       batch: {}
-
     exporters:
       debug: {}
       prometheusremotewrite:
         endpoint: {{ .Values.kof.metrics.endpoint }}
-        tls:
-          insecure: true
+        {{- include "kof-collectors.helper.tls_options" .Values.kof.metrics | indent 8 }}
         auth:
           authenticator: basicauth/metrics
       otlphttp:
         auth:
           authenticator: basicauth/logs
+{{- include "kof-collectors.helper.tls_options" .Values.kof.logs | indent 8 }}
```

Few more cases left from https://github.com/k0rdent/kof/pull/132#discussion_r1985360123

Please apply few suggestions below:
```suggestion
          authenticator: basicauth/logs
          {{- include "kof-collectors.helper.tls_options" .Values.kof.logs | indent 8 }}
```